_schedule/talks/2022-10-18-18-00-board-game-night.md

```diff
@@ -2,14 +2,15 @@
 abstract: Bring your board games, card games, and/or a sense of whimsy to the annual
   DjangoCon US Board Game Night!
 accepted: true
-category: social event
+category: social-event
 date: 2022-10-18 18:00:00-07:00
 difficulty: null
 end_date: 2022-10-18 22:00:00-07:00
+group: social-event
 layout: session-details
-schedule_layout: full
-permalink: /social-event/board-game-night/
+permalink: /social-events/board-game-night/
```

Comment:
I need to check if the board game night post links here

Reply:
I didn't see it referenced in the post.